packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -119,10 +139,10 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
     * Custom Roles granted through group membership
 
     As a result, if an admin was granted a standard Role that is limited to a single Group, and at the same time received group management privileges on all Groups in the org through a Custom Role, the ultimate outcome is group management on all Groups.
-2. During BETA, Custom Roles, Permissions and resources are limited to User and Group related items. We plan to introduce additional Resources and Permissions over time and appreciate your feedback.
-3. You can't assign a Custom Role without a Resource Set, hence always being applicable only to a subset of resources. Standard Roles on the other hand, are always initially granted to the entire org. They are only scoped to specific resources by subsequent invoking of the [target operations](/docs/reference/api/roles/#role-target-operations).
+3. <ApiLifecycle access="ea" />You can't assign a Custom Role without a Resource Set, hence always being applicable only to a subset of resources. Standard Roles on the other hand, are always initially granted to the entire org. They are only scoped to specific resources by subsequent invoking of the [target operations](/docs/reference/api/roles/#role-target-operations).
```

Comment:
should this be '2' now?  Also I realize the language didn't change, but maybe we should:  "can't" -> "cannot", and In the second sentence I would suggest changing ", are always initially granted to the entire org" into ", are always scoped to the entire org initially.".